fdbrpc/FlowTransport.actor.cpp

```diff
@@ -1211,7 +1211,7 @@ static void scanPackets(TransportData* transport,
 		++transport->countPacketsReceived;
 
 		if (packetLen > FLOW_KNOBS->PACKET_WARNING) {
-			TraceEvent(transport->warnAlwaysForLargePacket ? SevWarnAlways : SevWarn, "LargePacketReceived")
```

Comment:
I am in favour of this change, but it looks like we should completely deprecate the `warnAlwaysForLargePacket` field in this case.